lua/autorun/client/cl_RoleVote_vgui_list.lua

```diff
@@ -85,6 +103,25 @@ function PANEL:Init()
 	local buttonContainer = vgui.Create("DListLayout", scroll)
 	buttonContainer:Dock(FILL)
 
+	-- searchbar value change listener
+	function searchBar:OnValueChange(val)
+		-- check if val is empty
+		if val ~= "" then
+			-- filter buttons for val match
+			for _, child in ipairs(buttonContainer:GetChildren()) do
+				-- set visible to val find in name
+				child:SetVisible(string.find(child:GetName(), val))
```

Comment:
```suggestion
				local displayName = LANG.TryTranslation(child:GetName()):lower()
				child:SetVisible(displayName:find(val:lower()))
```
Search for the in the UI displayed name, not the internal name and ignore case